UefiTestingPkg/FunctionalSystemTests/MpManagement/Driver/MpManagement.inf

```diff
@@ -0,0 +1,82 @@
+## @file
+#  MP management driver that supports the power management of AP on/off and
+#  suspend/resume for all cores.
+#
+#  Copyright (c) 2022, Qualcomm Innovation Center, Inc. All rights reserved.<BR>
+#  Copyright (c) Microsoft Corporation.
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 1.29
+  BASE_NAME                      = MpManagement
+  FILE_GUID                      = 945DEFA8-8F4A-4049-BCFC-906D2B1D9455
+  MODULE_TYPE                    = DXE_DRIVER
+  VERSION_STRING                 = 0.1
+  ENTRY_POINT                    = MpManagementEntryPoint
+
+#
+# The following information is for reference only and not required by the build tools.
+#
+#  VALID_ARCHITECTURES           = IA32 X64 AARCH64

```

Comment:
If this is currently only supporting arm, should the IA32 X64 be removed for now?

Reply:
Agreed, will remove. thanks!

Reply:
Removed in the latest updated.